### ISSUE TITLE
ToggleSelection invisible when setting selection prefix

### ISSUE THREAD

**Issue by therockmandolinist** [peco/peco#388]
# What is the peco version, OS, architecture? 
```
peco: 0.4.8
OS:   macOS 10.12.13
```

# Abstract

When setting `SelectionPrefix` (after commit ff10112452fd692370292bc9ff464e1527f22725), `ToggleSelection` doesn't visually show selections when pressing `C-Space`. 

# Discussion

I'm thinking ToggleSelection should either:

- Ignore `SelectionPrefix` and continue to use colors for selecting items
- Add a prefix to selected items too to mark them as selected? This might be slightly more confusing, if consistent.

Or, it could use a different prefix char to mark selections? (To distinguish from currently selected item).

**Comment by lestrrat** [comment]
@therockmandolinist Ah, yeah, something like that was bound to happen. Oh well.

Can you conjure up some scheme to make this work? TBH I will not be using the selection-prefix feature myself, so it would be better for someone who wants to use it to come up with a plan.

**Comment by therockmandolinist** [comment]
> Can you conjure up some scheme to make this work?

@lestrrat could you elaborate on what you mean by that? Do you mean code, or just some conceptual solution?

**Comment by lestrrat** [comment]
@therockmandolinist some sort of spec, I guess.

**Comment by therockmandolinist** [comment]
@lestrrat I'm thinking the least confusing option is just using the `SavedSelection` face.

Otherwise, we have to deal with changing the char used for saved selections, and what to do if someone wants to use that char for their prefix, and what to do when prefix char is on one of those chars and it just seems like an unnecessary hassle.

**Comment by lestrrat** [comment]
@therockmandolinist oh wait, this might be a trivial bug.

Can you check bc7e5ea and see if this fixes it?

**Comment by therockmandolinist** [comment]
@lestrrat oh, neat. just built from source from that branch, seems to work:
<img width="572" alt="screen shot 2017-02-28 at 11 40 45" src="https://cloud.githubusercontent.com/assets/9020453/23414674/c7e77a22-fdaa-11e6-8536-154998ad7f2d.png">
Thanks!

**Comment by therockmandolinist** [comment]
Also, perhaps related to #345, it was a tad confusing that I had to run from `$GOPATH/src/github.com/peco/peco` (note the nested `peco`'s), instead of just being able to run from `$GOPATH/src/github.com/peco`. It makes sense with respect to the `peco` organization, but perhaps a note on the README about how it has to be `peco/peco` wouldn't hurt. 

In any case, I figured it out as a total go newbie (and perhaps that's the real reason it confused me), so it's probably not urgent and no harm done.
